test/test_collection.py

```diff
@@ -786,6 +788,26 @@ def test_insert_many(self):
         self.assertFalse(result.acknowledged)
         self.assertEqual(20, db.test.count_documents({}))
 
+    def test_insert_many_invalid(self):
+        db = self.db
+        db.test.drop()
```

Comment:
Let's remove this `db.test.drop()` since in this test it doesn't actually matter what the database looks like.